modules/retail/pages/retail-deploy-terminals-other.adoc

```diff
@@ -212,3 +212,94 @@ Then you can specify a second Salt state with the wireless details for use durin
 The boot phase supports only WPA2 PSK wireless configuration.
 Salt-managed production configuration supports all features supported by all major operating systems.
 ====
+
+
+
+== Deploy Terminals and Auto-Accept Keys
+
+You can configure {productname} to automatically accept the keys of newly deployed terminals.
+This is achieved using Salt grains.
+For more information about automatically accepting Salt grains, see https://docs.saltstack.com/en/latest/topics/tutorials/autoaccept_grains.html.
+
+[IMPORTANT]
+====
+Automatically accepting keys is less secure than manually checking and accepting keys.
+Only use this method on trusted networks.
+====
+
+
+To do this, you first need to configure Saltboot, then configure the {productname} Server.
+You can configure Saltboot to automatically sign grains once, or set it to always auto-sign.
+Alternatively, you can configure Saltboot during PXE boot using kernel parameters.
+
+
+.Procedure: Configuring Saltboot to Auto-Sign Once
+. On the branch server, create a configuration file called [path]``./etc/salt/minion.d/autosign-grains-onetime.conf``.
```

Comment:
```suggestion
. On the branch server, create a configuration file called [path]``/etc/salt/minion.d/autosign-grains-onetime.conf``.
```

Reply:
I'm not sure about this suggestion, but the current path looks strange.

Reply:
This is method 2 above.
The idea is to start in any working directory, create the directory structure and the file and append it to initrd.
It should not change anything in /etc on branch server.

The same procedure is already documented in wireless section above

So the path is correct.